src/components/Button.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function styleObj(wide, color) {
+  if (wide) {
+    return {
+      backgroundColor: color,
+      flexBasis: '50%',
+    };
+  }
+  return {
+    backgroundColor: color,
+    flexBasis: '25%',
+  };
+}
+
+export default function Button(props) {
+  const { name } = props;
+  const { color } = props;
+  const { wide } = props;
+  const btnStyle = styleObj(wide, color);
+  return <button type="button" style={btnStyle}>{name}</button>;
+}
```

Comment:
Since you are using destructuring to get the props, you could just destructure right there where you are declaring props. I.E = `export default function Button( { name, color, wide } )` would have the same effect and would look classier :clinking_glasses: